src/machine/machine_atsamd51.go

```diff
@@ -44,6 +44,9 @@ const (
 	PinTCCF          PinMode = PinTimerAlt
 	PinTCCG          PinMode = PinTCCPDEC
 	PinInputPulldown PinMode = 18
+	PinCAN           PinMode = 19
+	PinCAN0          PinMode = PinSDHC
+	PinCAN1          PinMode = PinCom
```

Comment:
This part is not necessary for atsamd51/atsame53, but I added it for the purpose of code uniformity.